pyanaconda/core/kickstart/version.py.j2

```diff
@@ -0,0 +1,29 @@
+#
+# Supported kickstart version.
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+{% if distro_release == "rawhide" %}
+from pykickstart.version import DEVEL as VERSION
+{% elif distro_name == "fedora" %}
+from pykickstart.version import F{$ distro_release $} as VERSION
+{% elif distro_name == "rhel" %}
+from pykickstart.version import RHEL{$ distro_release $} as VERSION
```

Comment:
For RHEL, this won't help you. You also have to go though the `commands.py` file line by line and change the commands there. I don't think we want that in templates.

Reply:
True. Then again, this does not hurt things, I think?